actions/send_email_workflow_status/action.yml

```diff
@@ -128,8 +267,18 @@ runs:
         echo "Email subject: [${email_subject}]"
         echo "Email body:" && cat ${body_file}
         echo ""
-
-        echo "Send email..."
-        echo "TODO IMPLEMENT Actual send email"
-        echo "Send email DONE"
+        # create environment variables from subject to pass to next step
+        echo "SUBJECT=$(echo $email_subject)" >> $GITHUB_ENV
```

Comment:
The additional $(....) seems not necessary.
Why not use directly :
`echo "SUBJECT=$email_subject" >> $GITHUB_ENV`